services/movie.py

```diff
@@ -6,6 +7,7 @@
 def get_movies(
     genres_ids: list[int] = None,
     actors_ids: list[int] = None,
+    title: str = None
```

Comment:
```suggestion
    title: Optional[str] = None
```
Because None by default

Reply:
FIx in all places(only your code)